packages/lib/src/h5web/toolbar/controls/ToggleGroup.module.css

```diff
@@ -5,7 +5,7 @@
 
 .btn {
   composes: btn from '../Toolbar.module.css';
-  padding: 0;
+  padding: 0 !important; /* FIX style ordering issue with Vite */
```

Comment:
Could you put the link to the vite issue somewhere ? In the linked issue or in the comment ?

Reply:
There isn't one for the ordering specifically. The existing issue is for `composes` and relative paths like `../../my-shared-module.css`, which we don't hit because of the ordering issue and my decision to switch to global utilities. I'm planning on opening an issue for the ordering specifically asap.